package.json

```diff
@@ -11,6 +11,12 @@
         "url": "https://github.com/sponsors/ljharb"
     },
     "main": "lib/index.js",
+    "exports": {
+        ".": {
```

Comment:
this needs the array fallback to the string form, or it won't work on node 13.0 or 13.1.

Reply:
Can you show me a documentation of this so I can implement it in the PR? I actually haven't heard about this.